compile_locally.sh

```diff
@@ -9,14 +9,13 @@ EMOJI="💈"
 # Detect the OS and architecture
 OS=$(uname -s | tr '[:upper:]' '[:lower:]')
 ARCH=$(uname -m)
-VERSION="v0.1.2"
 
 # The list of projects to be installed
-PROJECTS=("dymension" "dymension-relayer" "roller" "dymension-rdk")
-REPOS=("" "" "" "")
-VERSIONS=("v1.0.0-rc2" "" "v0.1.0" "v0.4.0-rc2")
-BUILDCOMMANDS=("" "" "" "make install_evm")
-BINARYNAME=("dymd" "rly" "roller" "rollapp_evm")
+PROJECTS=("dymension" "dymension-relayer" "dymension-rdk")
+REPOS=("" "" "")
+VERSIONS=("v1.0.0-rc2" "" "v0.4.0-rc2")
```

Comment:
can we make the versions it use github workflow environments so we won't have to hardcode it? 
https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Sure, but hardcoding gives us a clear git history of versions used - that's invaluable for debugging and rollback. Updating environment variables doesn't offer that. wdyt?

Reply:
I believe the workflow log will contain the relevant vars used so we can track that. but not critical for now.